go.mod

```diff
@@ -99,3 +99,5 @@ require (
 
 // FIXME the official repo has some bug makes br_gcs test failed. https://github.com/googleapis/google-cloud-go/pull/3509
 // replace cloud.google.com/go/storage => github.com/3pointer/google-cloud-go/storage v1.6.1-0.20210108125931-b59bfa0720b2
+
+replace github.com/pingcap/kvproto => github.com/MyonKeminta/kvproto v0.0.0-20211021102703-60266c23d5bd
```

Comment:
It doesn't matter much since we are merging this PR to the development branch. But since we decided to merge the PRs other than TiDB ones, I think that one should be merged first.